README.md

```diff
@@ -148,7 +148,7 @@ $ ng serve
  The source object has the following properties:
  * sourceType : avatar source ( Facebook, twitter, etc)
  * sourceId : identifier of the user
- * getAvatar(size?) : method to fetch user avatar from the current source
+ * getAvatar(size) : method to fetch user avatar from the current source
```

Comment:
I think this can't be required parameter.

Reply:
Yes it can, and must be, required. Because the only calls to that method are done by `AvatarComponent` which always supplies a non-null number (because there is a default value in the component). Almost all sources assumes it to be non-null, typically when concatenating directly into an URL.

Reply:
Yep, I'm all right with this since the other comment.